packages/zend-filter/library/Zend/Filter/Compress/Bz2.php

```diff
@@ -151,7 +151,8 @@ public function compress($content)
     public function decompress($content)
     {
         $archive = $this->getArchive();
-        if (@file_exists($content)) {
+        // check $content for NULL bytes or else file_exists will error out
+        if ((0 === preg_match('/\0/', $content)) && @file_exists($content)) {
```

Comment:
```suggestion
        if (strpos($content, '/\0/') === false && @file_exists($content)) {
```

use `strpos` and remote superfluous parens